docs/arcaflow/workflows/expressions.md

```diff
@@ -260,7 +260,7 @@ Functions:
 | `toLower(string)`                                | string       | Returns the input with Unicode letters mapped to their lower case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
-| `readFile(string)`                    | string | Returns a string representation of a file.<br>&nbsp; Param 1: The filepath string of the file to read into memory.<br>&nbsp;                                                                                     |
+| `readFile(string)`                    | string | Given a string containing a file path, returns the contents of the indicated file as a string.<br>&nbsp;                                                                                     |
 
```

Comment:
Now that the separate parameter description is gone, you should probably remove the trailing `<br>&nbsp;` so that we don't force a blank line at the bottom of the box.